<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComConfigWidget</class>
 <widget class="QWidget" name="ComConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="txsz_group">
     <property name="title">
      <string>通讯设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0" colspan="8">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>发送</string>
        </property>
        <widget class="QPushButton" name="modifyCommunicationSetup_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>621</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>修改发送设置</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>20</y>
           <width>621</width>
           <height>22</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="port_label">
            <property name="text">
             <string>端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="port_lineEdit">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="topic_label">
            <property name="text">
             <string>Topic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="topic_lineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="topic_label_2">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="id_lineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>停止位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>开启状态</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>波特率</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>数据位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="8">
       <widget class="QPushButton" name="modifyCommunicationSetup">
        <property name="text">
         <string>修改串口设置</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>校验位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_62">
        <property name="text">
         <string>地址</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="8">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QCheckBox" name="txsz_check3">
          <property name="text">
           <string>设备二</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal3"/>
        </item>
        <item row="4" column="6">
         <widget class="QLineEdit" name="addr5">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit6"/>
        </item>
        <item row="1" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal2"/>
        </item>
        <item row="5" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit6"/>
        </item>
        <item row="0" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit1"/>
        </item>
        <item row="6" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit7"/>
        </item>
        <item row="2" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit3"/>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="txsz_check5">
          <property name="text">
           <string>设备四</string>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QLineEdit" name="addr6">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit5"/>
        </item>
        <item row="3" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit4">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal1"/>
        </item>
        <item row="6" column="6">
         <widget class="QLineEdit" name="addr7">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QLineEdit" name="addr2">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal5"/>
        </item>
        <item row="4" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit5">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit1">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit6"/>
        </item>
        <item row="6" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal7"/>
        </item>
        <item row="2" column="6">
         <widget class="QLineEdit" name="addr3">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="txsz_check7">
          <property name="text">
           <string>设备六</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit4"/>
        </item>
        <item row="1" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit2">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal6"/>
        </item>
        <item row="3" column="1">
         <widget class="QComPortSelectedComboBox" name="txsz_chnal4"/>
        </item>
        <item row="1" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit2"/>
        </item>
        <item row="4" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit5"/>
        </item>
        <item row="2" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit3"/>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="txsz_check6">
          <property name="text">
           <string>设备五</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit7"/>
        </item>
        <item row="4" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit5"/>
        </item>
        <item row="2" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit3">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit1"/>
        </item>
        <item row="3" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit4"/>
        </item>
        <item row="6" column="2">
         <widget class="QComBaudrateSelectedComboBox" name="txsz_bit7"/>
        </item>
        <item row="6" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit7"/>
        </item>
        <item row="3" column="6">
         <widget class="QLineEdit" name="addr4">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QComDataBitSelectedComboBox" name="data_bit2"/>
        </item>
        <item row="2" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit3"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="txsz_check2">
          <property name="text">
           <string>设备一</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit2"/>
        </item>
        <item row="5" column="3">
         <widget class="QComParityBitSelectedComboBox" name="parity_bit6"/>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="txsz_check4">
          <property name="text">
           <string>设备三</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit4"/>
        </item>
        <item row="0" column="5">
         <widget class="QComStopBitSelectedComboBox" name="stop_bit1"/>
        </item>
        <item row="0" column="6">
         <widget class="QLineEdit" name="addr1">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="txsz_check1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>GPS</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>串口名</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QComPortSelectedComboBox</class>
   <extends>QComboBox</extends>
   <header>comdata/comselectedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QComBaudrateSelectedComboBox</class>
   <extends>QComboBox</extends>
   <header>comdata/comselectedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QComParityBitSelectedComboBox</class>
   <extends>QComboBox</extends>
   <header>comdata/comselectedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QComStopBitSelectedComboBox</class>
   <extends>QComboBox</extends>
   <header>comdata/comselectedcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QComDataBitSelectedComboBox</class>
   <extends>QComboBox</extends>
   <header>comdata/comselectedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
